admin/kernel-masterdata-service/src/test/java/io/mosip/kernel/masterdata/test/service/SchemaServiceTest.java

```diff
@@ -322,9 +321,17 @@ public void testDeleteIdentitySchemaFailed() throws Exception {
 	
 	@Test(expected = MasterDataServiceException.class)
 	@WithUserDetails("global-admin")
-	public void testDeleteIdentitySchemaFailedUpdate() throws Exception {		
-		Mockito.when(identitySchemaRepository.deleteIdentitySchema(Mockito.anyString(),  Mockito.any(LocalDateTime.class), 
-				Mockito.anyString())).thenThrow(DataAccessLayerException.class);		
+	public void testDeleteIdentitySchemaFailedUpdate() throws Exception {
+		Mockito.when(identitySchemaRepository.findIdentitySchemaById(
+				Mockito.anyString())).thenThrow(DataAccessLayerException.class);
+		identitySchemaService.deleteSchema("test-test");
+	}
+
+	@Test(expected = RequestException.class)
+	@WithUserDetails("global-admin")
+	public void testDeletePublishedIdentitySchema() throws Exception {
```

Comment:
Test method naming convention is not followed.
methodName_testCase_testResult

Reply:
taken care